index.html

```diff
@@ -0,0 +1,124 @@
+    <nav class="Lnav">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li class="camp">Campaign</li>
+      </ul>
+      <div class="menu_bar">
+        <i class="fas fa-bars"></i>
+      </div>
+    </nav>
+  </div>
+  <main>
+    <section class="first_sec">
+      <div>
+        <p class="wel">"Welcome to GLOBAL SCHOOL"</p>
+        <h1 class="title">
+          LET'S CONTRIBUTE TO THE DEVELOPMENT OF OUR COMMUNITY
+        </h1>
+        <p class="intro">
+          Join us in making a positive impact! Let's come together to contribute to the growth and betterment of our
+          community. Every effort counts, so let's take action now to ensure a brighter future for all. Let's contribute
+          to the development of our community!
+        </p>
+        <p class="hrs">2022.02.14(Mon) ~ 18(FRI)</p>
+        <p class="lieu">@ Greater Accra, Tersano police station</p>
+      </div>
+    </section>
+    <section class="snd_section">
+      <h2 class="Mpr">Main Program</h2>
+      <span class="dash"></span>
+      <div class="contain_program">
+        <div class="program">
+          <img src="images/lecture.png" alt="lecture's icon" />
+          <h3>Lecture</h3>
+          <p>Listen to the speakers from various countries.</p>
+        </div>
+        <div class="program">
+          <img src="images/forum.png" alt="lecture's icon" />
+          <h3>Exhibition</h3>
+          <p>Get access to more than 10,000 courses.</p>
+        </div>
+        <div class="program">
+          <img src="images/excibition.png" alt="lecture's icon" />
+          <h3>Forum</h3>
+          <p>Share your thoughts and views with a large community.</p>
+        </div>
+        <div class="program">
+          <img src="images/workshop.png" alt="lecture's icon" />
+          <h3>Location</h3>
+          <p>Global school is locate in more than 100 countries.</p>
+        </div>
+        <div class="program">
+          <img src="images/ignite.png" alt="lecture's icon" />
+          <h3>Organization</h3>
+          <p>Get great opportunities. And get assist and help from experts.</p>
+        </div>
+      </div>
+      <button type="button">JOIN OUR MEETING</button>
+    </section>
+    <section class="trd_session">
+      <h2>Featured Speakers</h2>
+      <article class="contain_speaker">
+        <!-- Speakers card-->
+      </article>
+      <button id="more" type="button">
+        More <i class="fas fa-caret-down"></i>
+      </button>
+      <article class="partner">
+        <h2>Partner</h2>
+        <div>
+          <i class="fab fa-google-pay fa-5x"></i>
+          <i class="fab fa-apple fa-5x"></i>
+          <i class="fab fa-supple fa-7x"></i>
+          <i class="fab fa-windows fa-5x"></i>
+          <i class="fab fa-github-square fa-5x"></i>
+        </div>
```

Comment:
- Kindly use the background color to stick to the Given design and have a better UI 👍
### Given Design 👇
![2023-04-25_13-27_1](https://user-images.githubusercontent.com/52010789/234260295-03cc5142-38c3-4b85-a9ea-395e0ae1192b.png)
### Your Design 👇
![2023-04-25_13-28](https://user-images.githubusercontent.com/52010789/234260303-4c55bc0b-05b3-4247-bae1-245518e60925.png)